README.md

```diff
@@ -55,6 +56,59 @@ $ docker run -P \
 --rm -v /etc/vsftpd/:/etc/vsftpd/ \
 --name vsftpd vsftpd
 ```
+
+
+## Extending using source-to-image
+
+You are able to extend this container image using [source-to-image](https://github.com/openshift/source-to-image):
+
+* supply a custom script which will be executed before running vsftpd
+* supply a configuration file on your own
+
+In order to extend the image using s2i, you need to prepare a directory with following structure:
+```
+$ tree default-conf
+default-conf
+├── init-hook
+└── vsftpd.conf
+```
+
+Both files are optional. But you should supply at least one, what would be the point of supplying neither, right?
+
+The file init-hook will be sourced using default shell. You can take a look at
+[`default-conf`](https://github.com/container-images/vsftpd/tree/master/default-conf)
+directory and use it:
+
+```
+$ git clone https://github.com/container-images/vsftpd && cd vsftpd
+
+$ sudo s2i build $PWD/default-conf modularitycontainers/vsftpd new-vsftpd
+I0915 14:46:16.578040 02356 install.go:251] Using "assemble" installed from "image:///usr/local/s2i/assemble"
+I0915 14:46:16.578125 02356 install.go:251] Using "run" installed from "image:///usr/local/s2i/run"
+I0915 14:46:16.578184 02356 install.go:251] Using "save-artifacts" installed from "image:///usr/local/s2i/save-artifacts"
+/opt/app-root/src
+'/tmp/src/init-hook' -> './init-hook'
+'/tmp/src/vsftpd.conf' -> './vsftpd.conf'
+```
+
+The files are copied into the new container image named `new-vsftpd`, we can run it now:
+
+```
+$ sudo docker run new-vsftpd
++ set -o nounset
++ set -o pipefail
++ '[' -f ./vsftpd.conf ']'
++ CONF_FILE=./vsftpd.conf
++ '[' -f ./init-hook ']'
++ source ./init-hook
+++ echo 'You can put your custom logic here'
+You can put your custom logic here
++ exec /usr/sbin/vsftpd ./vsftpd.conf
```

Comment:
When I tested it ```docker run``` wrote only output of the commands specified above, that means one line
```
You can put your custom logic here
```
Could you please take look at this?

Reply:
You are correct, the readme content is not up to date. I will update.